assets/src/domain/eventEditor/ui/datetimes/hooks/useDatesActionMenuHandler.tsx

```diff
@@ -12,31 +12,34 @@ import withIsLoaded from '@sharedUI/hoc/withIsLoaded';
 type DatesSubscriptionCallback = EntityActionsSubscriptionCb<Datetime, 'datetime'>;
 
 const useDatesActionMenuHandler = (): DatesSubscriptionCallback => {
-	return useCallback<DatesSubscriptionCallback>(({ entityType, entity: date }, { registerMenuItem }) => {
-		// although this is not needed
-		if (entityType !== 'datetime') {
-			return;
-		}
-		const withTicketsLoaded = withIsLoaded(TypeName.tickets);
+	return useCallback<DatesSubscriptionCallback>(
+		({ entityType, entity: date }, { registerElement: registerMenuItem }) => {
+			// although this is not needed
+			if (entityType !== 'datetime') {
+				return;
+			}
+			const withTicketsLoaded = withIsLoaded(TypeName.tickets);
 
-		registerMenuItem('editDate', () => <EditDateButton />);
+			registerMenuItem('editDate', () => <EditDateButton />, 11);
```

Comment:
maybe smth like:
```
registerMenuItem({
  name: 'editDate',
  component: EditDateButton,
  priority: 11
});
```

or similar verbose approach

Reply:
Yeah, those improvements can be added.